volume/struct.go

```diff
@@ -220,6 +220,10 @@ func (v *Volinfo) Nodes() []uuid.UUID {
 	var nodes []uuid.UUID
 
 	for _, b := range v.Bricks {
+		// This is a BUG. nodes[] should be a set (like in python)
+		// to avoid duplicates. Although not in production, there
+		// can be multiple bricks on the same machine and this will
+		// result nodes being added multiple times to the list.
```

Comment:
For example: For a plain distribute or plain replica(2) volume, if my bricks are `192.168.56.1:/export/brick1` and `192.168.56.1:/export/brick2`, the same node UUID will appear twice in the `nodes` list as `b.ID` will be the same for both bricks.

This results in the `startBricks` being run twice on the same node. Each run of `startBricks` will loop over list of bricks and start them.

Is my understanding correct ?